packaging/debian/postinst

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
I concur with @ellismg here, we're not using any bashims and using sh will give the community greater freedom to try installing the package in other places.